melanoma_phd/database/variable/VariableDynamicMixin.py

```diff
@@ -25,25 +24,24 @@ def __init__(
         self._required_ids = self.__extract_all_required_ids(
             required_ids=config.required_ids, required_variables=config.required_variables
         )
-        super().__init__(config=config)
+        super().__init__(config)
 
     @property
     def required_ids(self) -> List[str]:
         return self._required_ids
 
-    @abstractmethod
     def create_new_series(self, dataframe: pd.DataFrame) -> Optional[pd.Series]:
         self._check_required_ids(dataframe=dataframe)
         return None
```

Comment:
Yeah, I see... this method was created if we want to add a new column to database dataframe on the fly, but so far it's not used... But it could be so interesting to have it at some point for creating a new variable depending on other variable values, so I'd keep it so far.